src/View/Components/Fields/Input.php

```diff
@@ -27,9 +27,9 @@ public function __construct(
         public ?string $onChange = null,
         public ?string $onChangeAttribute = null,
         public ?string $prefix = null,
-        public ?int $min = null,
```

Comment:
I would prefer to use `null|int|float` for this instead.

Reply:
Sure, Can I change this in this PR?

Reply:
sure!

Reply:
done